_posts/2015-11-07-Creme-de-la-Pods.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Introducing: Creme de la Pods"
+author: orta
+categories: twitter cocoapodsfeed cocoapods
+---
+
+TL;DR: _You could always find out what was new with [@CocoaPodsFeed](https://twitter.com/cocoapodsfeed) and [feeds.cocoapods.org](https://feeds.cocoapods.org). Now you can see new CocoaPods that have a [QI](https://guides.cocoapods.org/making/quality-indexes.html) score of over 75 on [@CremeDeLaPods](https://twitter.com/cremedelapods)._
+
+Keeping on top of the community's work is a challenge, with ~13,500 CocoaPods available and roughly 20 new pods every day. We work on CocoaPods to encourage  [contributions to OSS](https://github.com/CocoaPods/CocoaPods#project-goals). A by-product of this is that if you follow the whole stream of new CocoaPods, it can get hard to distinguish between "my first library" and "paradigm changing."
```

Comment:
lol, that was no intentional, but yeah might work. I meant "paradigm shift" for the record